code/game/objects/items/weapons/tanks/watertank.dm

```diff
@@ -75,9 +75,9 @@
 		QDEL_NULL(noz)
 	return ..()
 
-/obj/item/watertank/attack_hand(mob/user as mob)
-	if(src.loc == user)
-		ui_action_click()
```

Comment:
ui_action_click is not being used anywhere on watertank? If not then it could be removed from the file.

Reply:
It is, it was just manually calling it which is an issue vs calling toggle mister